docs/user_guide/bank/essentials_requests.md

```diff
@@ -61,36 +68,38 @@ This brings up details of the Request including:
   - Quantity
     - If you are using [custom units](special_custom_units.md), those custom units will appear here.
   - Total Inventory (across all Storage Locations)
-At the bottom of the screen are buttons letting you start to fulfill the Request, or to cancel it.
-![Request view](images/essentials/requests/essentials_requests_view.png)
+    At the bottom of the screen are buttons letting you start to fulfill the Request, or to cancel it.
+    ![Request view](images/essentials/requests/essentials_requests_view.png)
 
 ## Fulfilling a Request
-To fulfill a Request, bring up the Request list by clicking on "Requests" in the left-hand menu,  then click on "view" beside the request,  then scroll to the bottom of that screen and click "Fulfill request".
+
+To fulfill a Request, bring up the Request list by clicking on "Requests" in the left-hand menu, then click on "view" beside the request, then scroll to the bottom of that screen and click "Fulfill request".
 That will bring you into a screen that allows you to specify the details for the distribution based on that Request -- you'll see a notice "request started".
-Fill in the remaining needed information.  The fields include:
-- Partner (It would be rare indeed to change this)
+Fill in the remaining needed information. The fields include:
+
+- Partner (determined by partner who created the request, not possible to change)
```

Comment:
This was the only line I changed manually, the rest are from my code editors auto formatting on save. I can remove the other changes if preferred, but I don't think they are bad changes.

Reply:
Now that we let banks enter on behalf of partners,  that's not quite accurate.

How about "(from the request, not editable)'